eav/queryset.py

```diff
@@ -282,3 +282,79 @@ def get(self, *args, **kwargs):
         the ``Manager`` get method.
         """
         return super(EavQuerySet, self).get(*args, **kwargs)
+
+
+    def order_by(self, *fields):
+        """
+        Django doesn't support arbitrary order-by clauses.
+        In such case, it is required to construct custom order-by
+        clause manually using Django's conditional expressions.
+        This will be slow, of course.
+        """
+        order_clauses = []
+        query_clause = self
+
+        for term in [t.split('__') for t in fields]:
+            if len(term) == 2:
+                # Retrive Attribute over which the ordering is performed.
+                attr = Attribute.objects.get(slug=term[1])
```

Comment:
What happens, if you do `order_by('eav__foo', 'other_related_model__bar')` or `order_by('other_related_model__bar')`?
Am I wrong, or do you asssume here that all `order_by` fields with double underscore maps to one of your attributes?

Also, what happens if you do `order_by('eav__nonexistent')` raising a `DoesNotExist` on `Attribute`?

Reply:
`order_by('eav__foo', 'other_related_model__bar')` - This is valid error, I'm fixing it.
`order_by('other_related_model__bar')` - Same as above.
`order_by('eav__nonexistent')` - This will result in a standard error (`AttributeError`), same behaviour as vanilla `order_by`.

Reply:
As to `order_by('eav__foo__bar')`, I'm simply leaving it unsupported. I'm sure no-one will use and supporting this would required extremely ugly, compliacted and unefficient SQL code.